app/controllers/application_controller/ci_processing.rb

```diff
@@ -1006,7 +1006,9 @@ def identify_record(id, klass = self.class.model)
       record = find_by_id_filtered(klass, from_cid(id))
     rescue ActiveRecord::RecordNotFound
     rescue => @bang
-      if @explorer
+      # TODO(lsmola) NetworkManager, Remove this when we have a full representation of the NetworkManager
+      record = ManageIQ::Providers::NetworkManager.where(:id => id).first
```

Comment:
I think the where is better here, since find will throw an exception if it will not find it